examples/annotation_import/basics.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>We really need to clean up the scoping of these imports but that is beyond the scope of this project. </p><p> </p><p>You don't need:</p><p><code>import labelbox</code> at the top though.  </p><p> </p><p>Also you can do convert:</p><pre class="ql-syntax" spellcheck="false">from labelbox.schema.ontology import OntologyBuilder, Tool
from labelbox import Client, LabelingFrontend, LabelImport, MALPredictionImport, MediaType
</pre><p>into:</p><pre class="ql-syntax" spellcheck="false">from labelbox import Client, LabelingFrontend, LabelImport, MALPredictionImport, MediaType, OntologyBuilder, Tool
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Apologies, I need to pay closer attention to the imports. </p>

Reply:
Don't worry about it, no one does :)